src/common/regions/types.ts

```diff
@@ -206,15 +218,18 @@ export interface MetroAreaObject extends RegionObject {
  * Metropolitan Statistical Areas
  */
 export class MetroArea extends Region {
+  public readonly states: State[];
+
   constructor(
     name: string,
     urlSegment: string,
     fipsCode: FipsCode,
     population: number,
-    public counties: County[],
-    public states: State[],
+    statesFips: FipsCode[],
+    public readonly countiesFips: FipsCode[],
```

Comment:
Can you add a comment explaining that we intentionally only store FIPS codes instead of `County` objects so that we can construct metro objects without loading all the county objects (or whatever makes sense to you to explain this)?